e2e-tests/gef/cypress/integration/submit-to-checker-mia.spec.js

```diff
@@ -46,7 +46,7 @@ context('Submit to UKEF as MIA', () => {
       cy.uploadFile('test.pdf', `${manualInclusion.url(applicationId)}/upload`);
       manualInclusion.uploadSuccess('test.pdf');
       automaticCover.continueButton().click();
```

Comment:
is this right here - the file is called `submit-to-checker-mia` but here we're using `automaticCover.continueButton()` ?

Reply:
I think the button is just pulling the `[data-cy="continue-button"]` tag. 
both the `manualInclusion` and `automaticCover` have that tag in

Reply:
OK cool would be good to make this clearer in the future, no need for this PR